src/array/without.ts

```diff
@@ -3,15 +3,19 @@ export {}; // needed for TS to realize this file can be imported
 declare global {
   interface Array<T> {
     without(element: T): Array<T>;
+    without(...elements: T[]): Array<T>;
   }
 
   interface ReadonlyArray<T> {
     without(element: T): Array<T>;
+    without(...elements: T[]): Array<T>;
```

Comment:
Just curious, but if the runtime can take `...elements` and have that work for both `remove(a)` and `remove(a, b)`, would the type system do the same, i.e. we could remove lines 5 and 10?

Even if it's not technically needed, fine leaving it if its more clear.

Reply:
yes, but leaving lines 5 and 10 gives better code hints

Reply:
:+1: